SQL/0000-00-01-Permission.sql

```diff
@@ -123,3 +123,18 @@ INSERT INTO `user_perm_rel` (userID, permID)
   ORDER BY p.permID;
 
 SELECT 'Menu import completed' as 'Status';
+
+-- permissions for each notification module
+DROP TABLE IF EXISTS `notification_modules_perm_rel`;
```

Comment:
This might conflict with https://github.com/aces/Loris/pull/2711. Let see which gets merged first